_includes/assets/sass/components/_internal-pages.scss

```diff
@@ -1,14 +1,24 @@
 .internal-page {
-    @include tbds-small {
-        width: 100%;
-        max-width: $tbds-breakpoint-small;
-        margin: auto;
-    }
+    width: 100%;
+
     h1 {
         font-size: 3rem;
     }
+    
     h2 {
         font-size: 1.75rem;
         line-height: 2.09rem;
     }
-}
\ No newline at end of file
+}
+
+.internal-page--measure {
+    width: 100%;
+
+    @include tbds-small {
+        width: 70%;
+    }
+
+    @include tbds-medium {
+        width: 60%;
+    }
+}
```

Comment:
I'm not sure why but this whole file is using 4 space indentation, instead of 2, like the rest of the files.